manifests/01-registry-credentials-request-alibaba.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-image-registry
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  secretRef:
+    name: installer-cloud-credentials
+    namespace: openshift-image-registry
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AlibabaCloudProviderSpec
+    statementEntries:
+    - effect: Allow
+      action:
+      - oss:CreateBucket
+      - oss:DeleteBucket
```

Comment:
```suggestion
      - oss:DeleteBucket
      - oss:PutBucket
```

Reply:
> @mtulio I updated the policy as below:
> 
> ```
> {
>     "Version": "1",
>     "Statement": [
>         {
>             "Effect": "Allow",
>             "Action": [
>                 "oss:PutBucket",
>                 "oss:DeleteBucket",
>                 "oss:PutBucketTags",
>                 "oss:GetBucketTags",
>                 "oss:PutBucketAcl",
>                 "oss:GetBucketAcl",
>                 "oss:PutBucketEncryption",
>                 "oss:GetBucketEncryption",
>                 "oss:PutBucketLifecycle",
>                 "oss:GetBucketLifecycle",
>                 "oss:GetBucketLocation",
>                 "oss:GetBucketInfo",
>                 "oss:ListBucket",
>                 "oss:GetObject",
>                 "oss:PutObject",
>                 "oss:DeleteObject",
>                 "oss:ListMultipartUploads",
>                 "oss:AbortMultipartUpload",
>                 "oss:ListParts"
>             ],
>             "Resource": "*"
>         }
>     ]
> }
> ```

@menglingwei it worked when I re-add the `"oss:CreateBucket"` action removed on last review.

``` shell
$ oc get configs.imageregistry.operator.openshift.io/cluster -n openshift-image-registry -o jsonpath='{.spec.storage}' |jq .
{
  "managementState": "Managed",
  "oss": {
    "bucket": "test-4627z-image-registry-us-east-1-pmilfulrtmxvagtworlwcvxkhu",
    "region": "us-east-1"
  }
}

$ oc patch configs.imageregistry.operator.openshift.io/cluster \
  --type='json' \
  -p='[{"op":"remove","path":"/spec/storage/oss"}]'

$ oc get configs.imageregistry.operator.openshift.io/cluster -n openshift-image-registry -o jsonpath='{.spec.storage}' |jq .
{
  "managementState": "Managed",
  "oss": {
    "bucket": "test-4627z-image-registry-us-east-1-mgbqjghgwpihreecpghsejhctm",
    "region": "us-east-1"
  }
}

```

Reply:
@menglingwei When I deployed with the latest settings for the credentials request I received the following error:
```
panic: Aliyun API Error: RequestId: 61B8C2F4FC21C632338BE36F Status Code: 403 Code: AccessDenied Message: The bucket you access does not belong to you.
```
Please add `oss:CreateBucket`. We need to get these figured out.

Thanks!